src/transformers/image_processing_utils.py

```diff
@@ -151,6 +151,11 @@ def center_crop(
             **kwargs,
         )
 
+    def to_dict(self):
+        encoder_dict = super().to_dict()
+        encoder_dict.pop("_valid_processor_keys", None)
```

Comment:
@molbap Would be good to have your confirmation here that we can remove - I don't think we want to save this out in the config: it's private and wouldn't be used when creating a new config class.

Reply:
yes, I agree! no use having it written to the config